packages/phone-number-privacy/package.json

```diff
@@ -18,6 +18,7 @@
     "db:migrate": "NODE_ENV=development ts-node scripts/runMigrations.ts"
   },
   "dependencies": {
+    "@celo/contractkit": "0.3.2-dev",
```

Comment:
We can only use published versions in cloud functions, so this needs to be 0.3.1